### PR TITLE
translate a few strings

### PR DESCRIPTION
**Pull request by salsakran**
This is going to take a while....

### CODE REVIEW

**Comment by joebordes** [comment]
Hi @salsakran 

I had a look at this change and I would like to offer my help. I think I can do this type of work. So if there are a lot of these files to review and you think it is plausible to delegate a batch of them I would be willing to give it a try.

**Comment by joebordes** [comment]
Do I just pick any files and start or may we duplicate our work if I do that? I mean I could do some files that someone else is working on

**Review by joebordes — COMMENTED** [review]
Hi @salsakran 

I created a [PR](https://github.com/metabase/metabase/pull/6248) which needs reviewing. I need a little help understanding how c3po works so, before going further, I wanted to ask you some doubts which I left on the PR

Thanks!

File: frontend/src/metabase/dashboards/components/DashboardList.jsx [hunk 1]
```diff
@@ -5,6 +5,8 @@ import PropTypes from "prop-types";
 import {Link} from "react-router";
 import cx from "classnames";
 import moment from "moment";
+import { t } from 'c-3po'
```
[the remark this answers is not in the document]
Comment (kdoh) [reply frontend/src/metabase/dashboards/components/DashboardList.jsx]: If I'm not mistaken `t` is actually [available as a global](https://github.com/metabase/metabase/blob/master/frontend/src/metabase/app.js#L8) so we shouldn't need to have to import it in every file.

**Comment by kdoh** [comment]
@salsakran Unless you have any objections I'm going to grab this, fix the issues / merge conflicts and hopefully get this ready to go with the changes as is just so we don't have this sit too long.

**Pull request merged**